Automattic/newspack-theme
# feat(woocommerce): remove woocommerce templates and dequeueing

### All Submissions:






### Changes proposed in this Pull Request:



Templates will be handled by Newspack Blocks (as of https://github.com/Automattic/newspack-blocks/pull/1561), dequeuing by Newspack Blocks (https://github.com/Automattic/newspack-blocks/pull/1572)

### How to test the changes in this Pull Request:

1. Switch `newspack-blocks` to `feat/dequeue-wc-styles` branch (https://github.com/Automattic/newspack-blocks/pull/1572)
1. Complete a transaction via the standard WC flow*, observe the flow uses standard WC templates (not the shortened ones)
3. Complete a transaction via the Donate block (with modal checkout), observe it uses customised templates (shortened) 

\* create a non-donation product and visit `/shop` page

### Other information:
